runtimes/pythonrt/local_runner_windows.go

```diff
@@ -0,0 +1,7 @@
+//go:build windows
+
+package pythonrt
+
+import "os/exec"
+
+func setCmdSysAttrPGID(cmd *exec.Cmd) {}
```

Comment:
Does this mean that on windows if `ak` fails the Python process will continue to run? @efiShtain ?

Reply:
+1. Maybe it's better to have a Windows version of `Start` that ensures the runner is shut down without PGID?